js-runtime/index.js

```diff
@@ -740,6 +740,8 @@ module.exports = {
   // Array opcodes TODO more to come
   newarr:   size => new Array(), // Ignored because JS push doesn't behave as desired
   pusharr: (arr, val, size) => arr.push(val),
+  pushf: (arr, val, size) => arr.push(val),
+  pushv: (arr, val, size) => arr.push(val),
```

Comment:
Should these opcodes even have a size argument?

Reply:
I'm not entirely sure, tbh. Won't `pushf` need it, at least? (to distinguish between the different-sized values, assuming we figure out associated problems). I can get rid of it in `pushv` though, I don't see a scenario where it's ever helpful

Reply:
Well, in that case we don't need the new opcodes? This was simply fixed vs variable sized and then we don't care about the 1, 2, 4, 8 differences.